core/state_processor.go

```diff
@@ -466,24 +470,22 @@ func (p *StateProcessor) Process(block *types.WorkObject, batch ethdb.Batch) (ty
 						quaiCoinbaseEtxs[coinbaseAddrWithLockup] = tx.Value() // this creates a new *big.Int
 					}
 				}
-				// subtract the minimum tx gas from the gas pool
-				if err := gp.SubGas(params.TxGas); err != nil {
-					return nil, nil, nil, nil, 0, 0, 0, nil, err
+				if block.NumberU64(common.ZONE_CTX) > params.TimeToStartTx {
+					// subtract the minimum tx gas from the gas pool
+					if err := gp.SubGas(params.TxGas); err != nil {
+						return nil, nil, nil, nil, 0, 0, 0, nil, err
+					}
+					*usedGas += params.TxGas
+					totalEtxGas += params.TxGas
 				}
-				*usedGas += params.TxGas
-				totalEtxGas += params.TxGas
 				timeDelta := time.Since(startTimeEtx)
 				timeCoinbase += timeDelta
 				continue
 			}
 			if etx.To().IsInQiLedgerScope() {
 				if etx.ETXSender().Location().Equal(*etx.To().Location()) { // Quai->Qi Conversion
 					lock := new(big.Int).Add(header.Number(nodeCtx), new(big.Int).SetUint64(params.ConversionLockPeriod))
-					primeTerminus := p.hc.GetHeaderByHash(header.PrimeTerminusHash())
-					if primeTerminus == nil {
-						return nil, nil, nil, nil, 0, 0, 0, nil, fmt.Errorf("could not find prime terminus header %032x", header.PrimeTerminusHash())
-					}
-					value := misc.QuaiToQi(primeTerminus.WorkObjectHeader(), etx.Value()) // convert Quai to Qi
+					value := etx.Value() // convert Quai to Qi
```

Comment:
The value was already converted so we should change this comment

Reply:
removed it